src/components/Button/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ButtonHTMLAttributes } from 'react'
+
+interface Props extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'type' | 'htmlType'> {
+  id?: string
+  block?: boolean
+  children?: React.ReactNode
+  className?: string
+  disabled?: boolean
+  htmlType?: 'reset' | 'submit'
+  onClick?: () => void
+  size?: 'small' | 'large'
+  icon?: React.ReactNode
+  type?: 'danger' | 'success' | 'outlined' | 'text' | 'primary' | 'secondary'
+  vCenter?: boolean
+  isSubmit?: boolean
+  isLoading?: boolean
+}
+
+export const Button = ({
+  className,
+  size = 'large',
+  type = 'primary',
+  vCenter,
+  children,
+  disabled,
+  isSubmit,
+  isLoading = false,
+  ...buttonProps
+}: Props) => {
+  return (
+    <button {...buttonProps} disabled={Boolean(disabled)} type={isSubmit ? 'submit' : 'button'}>
+      {children}
+    </button>
+  )
+}
+
+export default Button
```

Comment:
By the way, you can just use the button from chakra ui (https://chakra-ui.com/docs/components/form/button) instead of creating your own component from scratch, if you don't want to bother styling the components from scratch :D